### DISCUSSION TITLE
Wazuh agent dropping Auditd events

### DISCUSSION THREAD

**Discussion by smithomega34** [Security-Onion-Solutions/securityonion#6848] [Unsupported Versions]
Hi,

I setup the command monitoring for root:
$ more wazuh.rules 
-a exit,always -F euid=0 -F arch=b32 -S execve -k audit-wazuh-c
-a exit,always -F euid=0 -F arch=b64 -S execve -k audit-wazuh-c

I got his all to work and I am seeing events in audit.log

type=EXECVE msg=audit(1642028322.209:237218): argc=6 a0="ping" a1="-c" a2="1" a3="-W" a4="1" a5="google.com"
type=CWD msg=audit(1642028322.209:237218):  cwd="/etc/pki/tls"
type=PATH msg=audit(1642028322.209:237218): item=0 name="/usr/bin/ping"

I can then follow this to the wazuh-manager and I find the data in the archives.log

2022 Jan 12 22:58:51 (servername) 147.105.217.2->/var/log/audit/audit.log type=SYSCALL msg=audit(1642028342.422:237238): arch=c000003e syscall=59 success=yes exit=0 a0=8dd890 a1=8fd0d0 a2=826f60 a3=7fffffffd8e0 items=2 ppid=17176 pid=28782 auid=2108981 uid=0 gid=0 euid=0 suid=0 fsuid=0 egid=0 sgid=0 fsgid=0 tty=(none) ses=191 comm="ping" exe="/usr/bin/ping" subj=unconfined_u:unconfined_r:unconfined_t:s0-s0:c0.c1023 key="audit-wazuh-c" type=EXECVE msg=audit(1642028342.422:237238): argc=6 a0="ping" a1="-c" a2="1" a3="-W" a4="1" a5="google.com" type=CWD msg=audit(1642028342.422:237238):  cwd="/etc/pki/tls" type=PATH msg=audit(1642028342.422:237238): item=0 name="/usr/bin/ping" inode=1049220 dev=fd:01 mode=0100755 ouid=0 ogid=0 rdev=00:00 obj=system_u:object_r:ping_exec_t:s0 objtype=NORMAL cap_fp=0000000000003000 cap_fi=0000000000000000 cap_fe=0 cap_fver=2 type=PATH msg=audit(1642028342.422:237238): item=1 name="/lib64/ld-linux-x86-64.so.2" inode=263123 dev=fd:01 mode=0100755 ouid=0 ogid=0 rdev=00:00 obj=system_u:object_r:ld_so_t:s0 objtype=NORMAL cap_fp=0000000000000000 cap_fi=0000000000000000 cap_fe=0 cap_fver=0 type=PROCTITLE msg=audit(1642028342.422:237238): proctitle=70696E67002D630031002D57003100676F6F676C652E636F6D

So the data gets to the manager, see "root" user exec 'ping' BUT PROBLEM

I only get 3 messages per connection cycle in the archive.log. They are triggering rules and the alert.log is updated for on THREE events at a time.

I see no errors and have set logall and disable buffers. Still just "3" messages.

![image](https://user-images.githubusercontent.com/97640336/149236903-16aef96d-d4c9-4c93-86f1-eb64902afc48.png)

Is this a throttle for stopping log flood? Any thoughts as to how to get all the messages?

Thanks
-Dusty

**Comment by dougburks** [comment]
> They are triggering rules and the alert.log is updated for on THREE events at a time.

Which Wazuh alert is firing? 

What is the rule id?
